test/genode.sh

```diff
@@ -20,8 +20,9 @@ cd /genode
 git remote add jklmnn https://github.com/jklmnn/genode.git
 git fetch jklmnn
 git checkout b0d01103e20981d64a81b8494359c9606017e4bf
-git clone https://github.com/Componolit/ada-components.git /genode/repos/ada-components -b ada_interfaces_issue_69
+git clone https://github.com/Componolit/ada-components.git /genode/repos/ada-components
 cd /genode/repos/ada-components
+git checkout 7c3d94265e9e11fb4aafbba0619b81d1f59561ed
```

Comment:
I wonder, should we use tags instead of commit ids?

Reply:
I cannot see any advantage over using commit ids. Also tags might not be fine grained enough (unless we tag every commit which makes it pointless). I'd probably go for a tag when we move the code to master, but not for wip states.

Reply:
OK